core/SConscript.bootloader_emu

```diff
@@ -116,39 +117,14 @@ SOURCE_MOD += [
 
 if NEW_RENDERING:
     CPPDEFINES_MOD += ['NEW_RENDERING']
-    if TREZOR_MODEL in ('T',):
-        CPPDEFINES_MOD += ['DISPLAY_RGB565']
```

Comment:
The definition of `DISPLAY_RGB565` is inconsistent - it's only included in the emulator build, and I discovered that we don't use `DISPLAY_RGB565` anywhere in the code. However, we do use the `DISPLAY_MONO8` macro.

We have two options:
1. Eliminate `DISPLAY_RGB565` since it is not used.
2. Consistently define `DISPLAY_XXX` whenever the corresponding Rust feature `display_xxx` is defined. 

Currently, I have no preference. The second option might be better, although it offers no immediate benefit.

Reply:
Decided for 2, as we will soon need to distinguish RGB585 vs RGBA8888 variants in emulator driver: https://github.com/trezor/trezor-firmware/commit/847dfd5e3f5a65e95bb6b938a1fa18f24f0edb1e

Also DISC2 seem to have been missing these macros for new_rendering, so fixed it here https://github.com/trezor/trezor-firmware/commit/3efabfae14c879373add069d6a14cad9341c23bf